code/WorkInProgress/rewardsLocker.dm

```diff
@@ -622,6 +622,15 @@
 					H.set_clothing_icon_dirty()
 					succ = TRUE
 
+				else if (istype(M, /obj/item/clothing/head/helmet/space/captain))
+					var/prev = M.name
+					M.name = "\improper commander's space helmet"
```

Comment:
Use of `\improper` here might not quite be correct. "a commander's space helmet" seems fine to me?

Reply:
good idea, axed the \improper